pkg/policies/branch/branch.go

```diff
@@ -456,15 +550,18 @@ func getConfig(ctx context.Context, c *github.Client, owner, repo string) (*OrgC
 
 func mergeConfig(oc *OrgConfig, rc *RepoConfig, repo string) *mergedConfig {
 	mc := &mergedConfig{
-		Action:          oc.Action,
-		EnforceDefault:  oc.EnforceDefault,
-		EnforceBranches: oc.EnforceBranches[repo],
-		RequireApproval: oc.RequireApproval,
-		ApprovalCount:   oc.ApprovalCount,
-		DismissStale:    oc.DismissStale,
-		BlockForce:      oc.BlockForce,
+		Action:                oc.Action,
+		EnforceDefault:        oc.EnforceDefault,
+		EnforceBranches:       oc.EnforceBranches[repo],
+		RequireApproval:       oc.RequireApproval,
+		ApprovalCount:         oc.ApprovalCount,
+		DismissStale:          oc.DismissStale,
+		BlockForce:            oc.BlockForce,
+		RequireUpToDateBranch: oc.RequireUpToDateBranch,
+		RequireStatusChecks:   oc.RequireStatusChecks,
 	}
 	mc.EnforceBranches = append(mc.EnforceBranches, rc.EnforceBranches...)
+	mc.RequireStatusChecks = append(mc.RequireStatusChecks, rc.RequireStatusChecks...)
```

Comment:
See comment on definition, do we simply want to append here, or allow replacement (only with repo override allowed)

Reply:
Fixed in f902892e5f9bf4315541c3f48c04061c0648cd77